Cargo.toml

```diff
@@ -25,6 +25,7 @@ members = [
 ]
 
 [dependencies]
+base64 = "~0.12.3"
```

Comment:
Needed to base64-encode byte arrays as `blob`s.

Reply:
I was following suit with the existing dependencies here, but having re-read [the docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements) just now I do think that `^` is the way to go. I'll change all of the dependencies to `^` in a separate PR if you're alright with that.

> ^ dependencies (the default)?

What do you mean by 'the default'? I would think that no special operators at all (i.e. simply specifying a version like `"0.12.3"`) would be considered the default.

Reply:
I mean that with no sigil, it is a `^` dependency.

>The string "0.1.12" is a semver version requirement. Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.

Reply:
Huh! I didn't know that, thanks.